070-Integrations/025-drizzle.mdx

```diff
@@ -0,0 +1,63 @@
+---
+title: Drizzle with Xata
+navTitle: Drizzle
+keywords: ['orm', 'drizzle', 'typescript', 'query builder']
+description: "Use the Drizzle integration as a TypeScript ORM"
+slug: integrations/drizzle
+published: true
+---
+
+<Alert status="warning">
+  The integration of Drizzle with Xata is in active development and is considered experimental.
+</Alert>
+
+[Drizzle](https://github.com/drizzle-team/drizzle-orm) is a TypeScript ORM that is focused on type safety and closely follows the SQL syntax. 
+
+### Installation
+
+Install both Drizzle and `@xata.io/drizzle` with the Xata client. You can use the following package managers:
+
+<TabbedCode tabs={['npm', 'pnpm', 'yarn']}>
+
+```sh
+npm install drizzle-orm @xata.io/drizzle @xata.io/client
+```
+
+```sh
+pnpm add drizzle-orm @xata.io/drizzle @xata.io/client
+```
+
+```sh
+yarn add drizzle-orm @xata.io/drizzle @xata.io/client
+```
+
+### Usage 
+
+You can use Drizzle with a Xata, as demonstrated in the following examples:
+
+ ```tsx
+ import { pgTable, text } from 'drizzle-orm/pg-core';
+ import { drizzle } from '@xata.io/drizzle';
+ import { eq } from 'drizzle-orm';
+ import { getXataClient } from './xata';
+
+const xata = getXataClient();
+
+const drivers = pgTable("drivers", {
+  id: text("id").primaryKey(),
+  surname: text("surname"),
+  forename: text("forename"),
+  nationality: text("nationality"),
+});
+
+const db = drizzle(xata);
+
+const result = await db
+  .select({ surname: drivers.surname, forename: drivers.forename })
+  .from(drivers)
+  .where(eq(drivers.nationality, "Spanish"))
+  .execute();
+ ```
```

Comment:
````suggestion
```tsx
import { pgTable, text } from 'drizzle-orm/pg-core';
import { drizzle } from '@xata.io/drizzle';
import { eq } from 'drizzle-orm';
import { getXataClient } from './xata';

const xata = getXataClient();

const drivers = pgTable('drivers', {
  id: text('id').primaryKey(),
  surname: text('surname'),
  forename: text('forename'),
  nationality: text('nationality')
});

const db = drizzle(xata);

const result = await db
  .select({ surname: drivers.surname, forename: drivers.forename })
  .from(drivers)
  .where(eq(drivers.nationality, 'Spanish'))
  .execute();

```
````

Reply:
That's whitespace only, right?